examples/ember-cli/todomvc/app/templates/application.hbs

```diff
@@ -0,0 +1,29 @@
+<section id="todoapp">
+  <header id="header">
+    <h1>todos</h1>
+    <input type="text" id="new-todo" onkeydown={{action "createTodo"}} placeholder="What needs to be done?" autofocus>
+  </header>
+    {{outlet}}
+    {{#if model.length}}
+      <footer id="footer">
+        <span id="todo-count"><strong>{{remaining.length}}</strong> {{pluralize 'item' remaining.length}} left</span>
+        <ul id="filters">
+          <li>{{#link-to "index" activeClass="selected"}}All{{/link-to}}</li>
+          <li>{{#link-to "active" activeClass="selected"}}Active{{/link-to}}</li>
+          <li>{{#link-to "completed" activeClass="selected"}}Completed{{/link-to}}</li>
+        </ul>
+        {{#if completed.length}}
+          <button id="clear-completed" {{action "clearCompleted"}}>Clear completed</button>
```

Comment:
It looks like the closure action syntax is used above for `onkeydown`. Since this action is on the controller as well, using that syntax here seems preferable:

```hbs
<button id="clear-completed" onclick={{action "clearCompleted"}}>Clear completed</button>
```